front/pages/api/stripe/webhook.ts

```diff
@@ -243,6 +244,17 @@ async function handler(
                 workspaceSeats: workspaceSeats,
               });
             }
+            const auth = await Authenticator.internalAdminForWorkspace(
```

Comment:
I thiink the event `checkout.session.completed` is only when you enter the initial checkout session to create a new subscription. Looking at the code line 402 it seems that this should be done in `customer.subscription.updated"

Reply:
Ok trusting you on this one (good thing I picked you for the review).
Updated !

Reply:
I'm actually trusting the code comments from PR who handled the reactivation 😄

Reply:
So it turns out we need both. When the subscription is completely cancelled, the user can get a new checkout session.